### PR TITLE
 :bug: Update check_pre_req.sh

### LINKED ISSUE #2245: :bug: Make dependency help missing in check_pre_req.sh
### Describe the bug

The Make utility does not have a "help" section in the `check_pre_req.sh` script. 

### Steps To Reproduce

1. Run `bash check_pre_req.sh -V go make kflex`.
2. Wait for installation checks to complete.
3. Review outputs.
4. See GNU Make is not followed by install suggestion.

No description seen:
image

### Expected Behavior

GNU Make should be followed by `sudo apt-get install make` or `sudo yum install make`.

image

### Additional Context

GNU Make was not included in WSL Ubuntu-24.02 or 20.04.

### PR DESCRIPTION
**Pull request by algoritmoalex**
## Summary
Added installation suggestion for GNU Make utility.

## Related issue(s)

Fixes #2245 

image

### CODE REVIEW

**Comment by ezrasilvera** [comment]
@algoritmoalex Thanks for the PR. Please squash the commits into a single commit. Please sign the commit

**Comment by ezrasilvera** [comment]
@algoritmoalex there are many guides on how to squash e.g., https://www.freecodecamp.org/news/git-squash-commits/  if you need help - please feel free to post to the KS slack channel

File: hack/check_pre_req.sh [hunk 1]
```diff
@@ -122,7 +122,8 @@ is_installed_kubectl() {
 is_installed_make() {
     is_installed 'GNU Make' \
         'make' \
-        'make --version | head -1'
+        'make --version | head -1' \
+        'sudo apt-get install make or sudo yum install make'
```
Comment (MikeSpreitzer) [inline hack/check_pre_req.sh:126]: That is not going to work on a Mac.

File: hack/check_pre_req.sh [hunk 2]
```diff
@@ -122,7 +122,8 @@ is_installed_kubectl() {
 is_installed_make() {
     is_installed 'GNU Make' \
         'make' \
-        'make --version | head -1'
+        'make --version | head -1' \
+        'sudo apt-get install make or brew install make'
```
Comment (MikeSpreitzer) [inline hack/check_pre_req.sh:126]: The situation on a Mac is more complicated. Developers generally need the Xcode Command Line Tools, which include a version of `make`. Additionally developers can install more advanced versions of various tools. I have chosen not to, and that is working for me.

**Comment by MikeSpreitzer** [comment]
The git history here is much more complicated than it needs to be, for this simple change. I recommend squashing all the commits together.

Also, see https://docs.google.com/document/d/1iwkhpoV76-J-3G-Rif4B1cmL9SWBO_f_fE6WK8jvsDc .

**Pull request closed without merging**